lib/passage_flutter_models/passage_app_info.dart

```diff
@@ -16,6 +17,7 @@ class PassageAppInfo implements PassageFlutterModel {
   final bool requireIdentifierVerification;
   final int sessionTimeoutLength;
   final List<PassageAppUserMetadataSchema>? userMetadataSchema;
+  final AuthMethods authMethods;
```

Comment:
We'll need to make this optional, as this is sometimes null.

Reply:
Never mind, it sounds like this is currently nullable but will NOT be in the near future.